### PR TITLE
adding @static method for all functions of sections 6,7,8

### PR DESCRIPTION
**Pull request by Nikhil008**

### CODE REVIEW

File: app/utils/common/is800_2007.py [hunk 1]
```diff
@@ -1377,14 +1415,15 @@ def cl_8_4_shear_buckling_coeff_Kv(only_at_support,c=None,d=None):
         Note:
               Reference - IS800_2007 cl.8.4.2.1 and cl.8.4.2.2
         """
-        if only_at_support == True:
+        if only_at_support:
```
Comment (ajmalbabums) [inline app/utils/common/is800_2007.py:1418]: check this

File: app/utils/common/is800_2007.py [hunk 2]
```diff
@@ -1401,7 +1440,7 @@ def cl_8_4_2_shear_buckling_check(d, t_w, k_v, fy, stiffeners):
                 IS 800:2007, cl. 8.4.2.1
         """
         epsilon = math.sqrt(250/fy)
-        if stiffeners == False:
+        if not stiffeners:
```
Comment (ajmalbabums) [inline app/utils/common/is800_2007.py:1443]: check this

**Review by ajmalbabums — COMMENTED** [review]
Check the comments

File: app/utils/common/is800_2007.py (hunk 1 above)
Comment (Nikhil008) [reply app/utils/common/is800_2007.py:1418]: yes == True is not required. only_at_support takes boolean values

**Pull request merged**